third_party/cert-manager/Makefile

```diff
@@ -10,7 +10,7 @@ deploy: $(TOOLBIN)/kubectl $(TOOLBIN)/helm
 	$(TOOLBIN)/kubectl create namespace cert-manager
 	$(TOOLBIN)/helm repo add jetstack https://charts.jetstack.io
 	$(TOOLBIN)/helm repo update
-	$(TOOLBIN)/helm install cert-manager jetstack/cert-manager --version v1.2.0 \
+	$(TOOLBIN)/helm install cert-manager jetstack/cert-manager --version v1.6.2 \
```

Comment:
This task in pipeline/pipeline.yaml should be updated I think.

  - name: upgrade-helm-certmgr-from-repo
    params:
    - name: helm_repo
      value: https://charts.jetstack.io
    - name: chart_name
      value: jetstack/cert-manager --create-namespace
    - name: release_version
      value: v1.2.0
    - name: release_name
      value: cert-manager
    - name: release_namespace
      value: cert-manager
    - name: overwrite_values
      value: installCRDs=true
    - name: image
      value: $(params.helm_image)
    taskRef:
      kind: Task
      name: helm-upgrade-from-repo
    workspaces:
    - name: source
      workspace: shared-workspace
    when:
    - Input: $(params.deployCertManager)
      Operator: in
      Values:
      - "true"
    - Input: $(params.IMAGE_DIGEST)
      Operator: in
      Values:
      - "none"
    - Input: $(params.INCREMENTAL_RESTART)